medic/views.py

```diff
@@ -1,89 +1,99 @@
 from medic.models import Medic
-from medic.serializers import MedicSerializer
-from medic.serializers import LoginSerializer
-from rest_framework import generics
-from rest_framework import permissions
-from medic.permissions import IsOwnerOrReadOnly
-from rest_framework.response import Response
-from rest_framework.exceptions import AuthenticationFailed
 from rest_framework.reverse import reverse
 import jwt, datetime
 from django.contrib.auth.hashers import check_password
+from django.views.generic.edit import FormView
+from medic.forms import MedicForm
+from django.shortcuts import render, redirect
+
+
+
+def register(request):
+    if request.method == 'POST':
+        form = MedicForm(request.POST)
```

Comment:
From here, you can tell if the form variable is holding data submitted on the form, and if you're actually saving the data. You can print it to know what it's holding when the form is submitted ...

Reply:
okay...let me do that